mpas_analysis/ocean/time_series_antarctic_melt.py

```diff
@@ -459,9 +460,9 @@ def run_task(self):  # {{{
         observationsDirectory = build_obs_path(config, 'ocean',
                                                'meltSubdirectory')
         obsFileNameDict = {'Rignot et al. (2013)':
-                           'Rignot_2013_melt_rates.csv',
+                           'Rignot_2013_melt_rates_20200623.csv',
                            'Rignot et al. (2013) SS':
-                           'Rignot_2013_melt_rates_SS.csv'}
+                           'Rignot_2013_melt_rates_SS_20200623.csv'}
```

Comment:
These should have had date stamps a long time ago.  I just copied them, updated the names of the Ross regions, and put on today's date stamp.